causal_testing/testing/causal_test_outcome.py

```diff
@@ -1,93 +1,7 @@
 from abc import ABC, abstractmethod
-from typing import Any, Union
-
+from causal_test_result import CausalTestResult
```

Comment:
This import needs to be changed to `from causal_testing.testing.causal_test_result import CausalTestResult`.

Reply:
Fixed